azure-pipelines.yml

```diff
@@ -0,0 +1,78 @@
+jobs:
+- job: Build
+  pool:
+    vmImage: ubuntu-16.04
+  container:
+    image: kphoenix/ci:1
+    options: --cap-add=SYS_PTRACE
+  steps:
+  - script: /root/scripts/azure-build.sh
+
+  - task: PublishPipelineArtifact@0
+    inputs:
+      artifactName: "build_archive"
+      targetPath: ./build.tar.gz
+
+- job: Lint
+  pool:
+    vmImage: ubuntu-16.04
+  container:
+    image: angr/ci:1
+    options: --cap-add=SYS_PTRACE
+  dependsOn: Build
+  condition: succeeded()
+  steps:
+  - task: DownloadPipelineArtifact@0
+    inputs:
+      artifactName: "build_archive"
+      targetPath: .
+
+  - script: /root/scripts/azure-lint.sh
+
+- job: Test
+  pool:
+    vmImage: ubuntu-16.04
+  container:
+    image: angr/ci:1
+    options: --cap-add=SYS_PTRACE
+  dependsOn: Build
+  condition: succeeded()
+  variables:
+    NUM_WORKERS: 9
```

Comment:
I don't think we needed to bump this down to 9 - the lint task should take short enough that it won't really matter if it causes us to hit our cpu cap

Reply:
I've reverted this for now, for now you're definitely correct. I might even be able to break it out so it uses the checked out version from azure and therefore can run in parallel to the build rather than the tests, further eliminating this need.